### PR TITLE
sci-libs/libxc: Bump to 5.1.0

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sci-libs/libxc/libxc-5.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		--disable-static \
+		$(use_enable fortran)
+}
+
+## Upstream recommends not running the test suite because it requires
+## human expert interpretation to determine whether output is an error or
+## expected under certain circumstances. Nevertheless, experts might want the option.
+## The autotools src_test function modified not to die. Runs emake check in build directory.
+src_test() {
+	debug-print-function ${FUNCNAME} "$@"
```
Comment (SoapGentoo) [inline sci-libs/libxc/libxc-5.1.0.ebuild:39]: remove, not useful here

File: sci-libs/libxc/libxc-5.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+	pushd "${BUILD_DIR}" > /dev/null || die
+	make check || ewarn "Make check failed. See above for details."
```
Comment (SoapGentoo) [inline sci-libs/libxc/libxc-5.1.0.ebuild:42]: `emake`?

File: sci-libs/libxc/libxc-5.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+## Upstream recommends not running the test suite because it requires
+## human expert interpretation to determine whether output is an error or
+## expected under certain circumstances. Nevertheless, experts might want the option.
+## The autotools src_test function modified not to die. Runs emake check in build directory.
+src_test() {
+	debug-print-function ${FUNCNAME} "$@"
+
+	pushd "${BUILD_DIR}" > /dev/null || die
```
Comment (SoapGentoo) [inline sci-libs/libxc/libxc-5.1.0.ebuild:41]: you're not building out of source are you?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-libs/libxc/libxc-5.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	make check || ewarn "Make check failed. See above for details."
+	einfo "emake check done"
+	popd > /dev/null || die
```
Comment (SoapGentoo) [inline sci-libs/libxc/libxc-5.1.0.ebuild:44]: in fact, all of this overriding seems pointless

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-libs/libxc/libxc-5.1.0.ebuild (hunk 4 above)
Comment (jsmolic) [reply sci-libs/libxc/libxc-5.1.0.ebuild:44]: I agree, I'll drop this altogether

Comment (jsmolic) [reply sci-libs/libxc/libxc-5.1.0.ebuild]: Removed

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**